src/client/app/shared/card/card.ts

```diff
@@ -0,0 +1,7 @@
+class Card {
+  constructor(x: number, y: number, radius: number) {
+    this.x = x || 0;
```

Comment:
I'm about to push our upgrade to TypeScript 2.0, which will include me turning on --[strictNullChecks](https://news.ycombinator.com/item?id=11325653).

These "truthy" checks will no longer be necessary unless the type explicitly shows " | null".